examples/cnsga/cnsga_tnk.py

```diff
@@ -0,0 +1,63 @@
+from xopt.cnsga import cnsga
```

Comment:
Is this just the notebook copied into a .py file? I'd rather just have a notebook.